tox.ini

```diff
@@ -1,26 +1,23 @@
 [tox]
 envlist =
-    py{35,36,37,38,39}-django22
-    py{36,37,38,39}-django30
-    py{36,37,38,39}-django31
+    py{36,37,38,39}-django{22,31,32}
```

Comment:
I think it still makes sense to test django 2.2 against python 3.5, as it officially supported in this version https://docs.djangoproject.com/en/2.2/releases/2.2/#python-compatibility